wagtailsharing/views.py

```diff
@@ -5,8 +5,8 @@
 
 from wagtail.contrib.routable_page.models import RoutablePageMixin
 from wagtail.core import hooks
-from wagtail.core.url_routing import RouteResult
-from wagtail.core.views import serve as wagtail_serve
+from wagtail.url_routing import RouteResult
+from wagtail.views import serve as wagtail_serve
```

Comment:
I would wrap this in 

try:
    from wagtail.url_routing import RouteResult
    from wagtail.views import serve as wagtail_serve
except ImportError:
    from wagtail.core.url_routing import RouteResult
    from wagtail.core.views import serve as wagtail_serve
```

for compatibility with pre 3.0 too

Reply:
Thanks @zerolab I'll try it this way